scripts/cargo-build.sh

```diff
@@ -2,4 +2,4 @@
 
 export WASM_BUILD_TOOLCHAIN=nightly-2021-02-20
 
-cargo +nightly-2021-02-20 build --release
+cargo +nightly-2021-02-20 build --release --features runtime-benchmarks
```

Comment:
Do we need this change?

Reply:
I would say yes. I remember finding it on the old `olympia` branch when I was merging. But I didn't keep it. I suppose now is a good time to add it back since for almost every runtime PR from now on we would need to benchmark relavant extrinsics

Reply:
Ok.